var/spack/repos/builtin/packages/damask-grid/package.py

```diff
@@ -32,14 +32,15 @@ class DamaskGrid(CMakePackage):
     depends_on("petsc@3.16.5:3.16", when="@3.0.0-alpha6")
     depends_on("petsc@3.14.0:3.14,3.15.1:3.16", when="@3.0.0-alpha5")
     depends_on("petsc@3.14.0:3.14,3.15.1:3.15", when="@3.0.0-alpha4")
-    depends_on("pkgconfig", type="build")
+    depends_on("pkg-config", type="build")
```

Comment:
I think this was correct before (`pkgconfig` is the virtual, and `pkg-config` and `pkgconf` are two packages providing it). Or is this a special case where only `pkg-config` works?

Reply:
I remember vaguely getting an build error due to missing `pkgconfig`. But I couldn't reproduce it.